code/src/java/pcgen/rules/persistence/util/TokenFamily.java

```diff
@@ -280,4 +283,24 @@ public void addDeferredToken(DeferredToken<?> newToken)
 	{
 		deferredTokenList.add(newToken);
 	}
+
+	public void addGroupDefinition(GroupDefinition<?> def)

```

Comment:
Added comments to GroupDefinition to hopefully explain it better.  If not, please let me know